### ISSUE TITLE
Creating an Enrollment Agent certificate for typical Service Acconts

### ISSUE THREAD

**Issue by SquirrelAssassin** [MicrosoftDocs/windows-itpro-docs#3792]
Under Creating an Enrollment Agent certificate for typical Service Acconts, you have "In the Certificate Template console, right-click the Exchange Enrollment Agent template in the details pane and click Duplicate Template."  I don't have a certificate template called "Exchange Enrollment Agent" I do have one called "Exchange Enrollment Agent (offline request)".  Should I have a template called "Exchange Enrollment Agent" or is the "Exchange Enrollment Agent (offline request)" the one you mean?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fd063d0c-8739-15bd-4f16-11b893029845
* Version Independent ID: cd3679fe-dd9c-44f9-04ed-c772a47ca4e4
* Content: [Configuring Hybrid Windows Hello for Business - Public Key Infrastructure (PKI)](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-pki)
* Content Source: [windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-pki.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-pki.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @mapalko
* Microsoft Alias: **mapalko**

**Comment by e0i** [comment]
@officedocsbot assign @e0i

**Comment by e0i** [comment]
@SquirrelAssassin 

The issues section of this repository is intended for product documentation issues only.

Please consider opening a product support ticket by the following link below for your question so that it gets resolved quickly.

[Windows 10 Support](https://support.microsoft.com/en-us/hub/4338813/windows-help?os=windows-10)

Thank you.

**Comment by MamaDoc** [comment]
Thank you

**Comment by SquirrelAssassin** [comment]
Let me rephrase my comment there is no such thing as a exchange enrollment agent it is exchange update enrollment (offline request). I was just trying to be nice. you need to update your documentation.

**Comment by SquirrelAssassin** [comment]
Here is a screenshot http://playinterchange.com/wp-content/uploads/2018/07/exchange-server-certificate-template-create-a-certificate-template-from-a-server-2012-r2-ca-mrchiyo-template.png

**Comment by illfated** [comment]
Please note: I only linked the PR here, just as I am linking this issue back to PR #4588 now.
@SteveBurkettNZ is author of the PR.

**Comment by e0i** [comment]
@SquirrelAssassin 

With the help of your feedback, the content has been updated by the community with relevant changes accordingly.

Thank you for helping to improve the Docs.

**Comment by e0i** [comment]
@officedocsbot close
